contracts/overseer/src/contract.rs

```diff
@@ -51,6 +56,16 @@ pub fn instantiate(
         },
     )?;
 
+    store_dynrate_config(
```

Comment:
in practice this stuff will be migrated, and not instantiated. lets create a `migrate` hook, and have the migrate hook store dynrate config and state from the `MigrateMsg`.

Reply:
also, lets consider pruning state -- don't think `DynrateState.update_vector` or `DynrateState.rate_delta` are used anywhere anymore

Reply:
done